LayoutTests/fast/forms/select-ask-for-reset.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<body>
+<script src="../../resources/testharness.js"></script>
+<script src="../../resources/testharnessreport.js"></script>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/forms.html#ask-for-a-reset">
+<div id="log"></div>
+<script>
```

Comment:
Some of these commits: https://github.com/chromium/chromium/commits/77578ccb4082ae20a9326d9e673225f1189ebb63/third_party/WebKit/LayoutTests/fast/forms/select/select-ask-for-reset.html?browsing_rename_history=true&new_path=third_party/blink/web_tests/fast/forms/select/select-ask-for-reset.html&original_branch=main 

might make the test that you removed pass.

Reply:
Yes - I am planning to work on them one by one. It is just my skills level restricting me to take big patches.